meta-balena-common/recipes-core/os-config/os-config_1.1.4.bb

```diff
@@ -8,7 +8,7 @@ inherit cargo
 # how to get os-config could be as easy as but default to a git checkout:
 # SRC_URI += "crate://crates.io/os-config/1.1.4"
 SRC_URI += "git://git@github.com/balena-os/os-config.git;protocol=ssh;nobranch=1"
-SRCREV = "1b8d0cef620e3007a807ccfc0be17bec15d09f9c"
+SRCREV = "46048edd57f3dbe4c03f192f28e85e6a7f1aaca7"
```

Comment:
@klutchell would you mind doing the update and the renaming in different commits? We want the ability to revert any of them independently I would think.